presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/PickTableLayout.java

```diff
@@ -60,10 +62,12 @@
 public class PickTableLayout
```

Comment:
All below are also using `binding -> true`:
 - `com.facebook.presto.sql.planner.optimizations.BeginTableWrite.Rewriter#rewriteDeleteTableScan`
 - `com.facebook.presto.sql.rewrite.ShowStatsRewrite.Visitor#getConstraint`
 - `com.facebook.presto.cost.CoefficientBasedStatsCalculator.Visitor#getConstraint`

Code related to `getTableLayout` and is a bit complicated. I think it would be nice to refactor this so other places could also leverage this.